2025-02-17 13:59:45,621 INFO    MainThread:59831 [wandb_init.py:setup_run_log_directory():637] Logging user logs to /Users/syedzaidi/Desktop/untitled folder 3/wandb-self-hosted/scripts/wandb/run-20250217_135945-dfc1he0n/logs/debug.log
2025-02-17 13:59:45,621 INFO    MainThread:59831 [wandb_init.py:setup_run_log_directory():638] Logging internal logs to /Users/syedzaidi/Desktop/untitled folder 3/wandb-self-hosted/scripts/wandb/run-20250217_135945-dfc1he0n/logs/debug-internal.log
2025-02-17 13:59:45,621 INFO    MainThread:59831 [wandb_init.py:init():756] calling init triggers
2025-02-17 13:59:45,621 INFO    MainThread:59831 [wandb_init.py:init():761] wandb.init called with sweep_config: {}
config: {'_wandb': {'code_path': 'code/scripts/log_runs.py'}}
2025-02-17 13:59:45,621 INFO    MainThread:59831 [wandb_init.py:init():789] starting backend
2025-02-17 13:59:45,621 INFO    MainThread:59831 [wandb_init.py:init():793] sending inform_init request
2025-02-17 13:59:45,623 INFO    MainThread:59831 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-02-17 13:59:45,623 INFO    MainThread:59831 [wandb_init.py:init():808] backend started and connected
2025-02-17 13:59:45,626 INFO    MainThread:59831 [wandb_init.py:init():901] updated telemetry
2025-02-17 13:59:45,654 INFO    MainThread:59831 [wandb_init.py:init():936] communicating run to backend with 90.0 second timeout
2025-02-17 13:59:45,987 INFO    MainThread:59831 [wandb_init.py:init():994] starting run threads in backend
2025-02-17 13:59:46,022 INFO    MainThread:59831 [wandb_run.py:_console_start():2385] atexit reg
2025-02-17 13:59:46,023 INFO    MainThread:59831 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-02-17 13:59:46,023 INFO    MainThread:59831 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-02-17 13:59:46,023 INFO    MainThread:59831 [wandb_run.py:_redirect():2325] Redirects installed.
2025-02-17 13:59:46,024 INFO    MainThread:59831 [wandb_init.py:init():1036] run started, returning control to user process
2025-02-17 13:59:46,024 INFO    MainThread:59831 [wandb_run.py:_finish():2110] finishing run ahsanmx518/POC-Project/dfc1he0n
2025-02-17 13:59:46,024 INFO    MainThread:59831 [wandb_run.py:_atexit_cleanup():2350] got exitcode: 0
2025-02-17 13:59:46,025 INFO    MainThread:59831 [wandb_run.py:_restore():2332] restore
2025-02-17 13:59:46,025 INFO    MainThread:59831 [wandb_run.py:_restore():2338] restore done
2025-02-17 13:59:47,659 INFO    MainThread:59831 [wandb_run.py:_footer_history_summary_info():3963] rendering history
2025-02-17 13:59:47,659 INFO    MainThread:59831 [wandb_run.py:_footer_history_summary_info():3995] rendering summary
2025-02-17 13:59:47,659 INFO    MainThread:59831 [wandb_run.py:_footer_sync_info():3924] logging synced files
